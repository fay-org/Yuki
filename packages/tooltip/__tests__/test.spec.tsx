import { mount } from '@vue/test-utils'
import { Tooltip } from '..'

describe('Tooltip', () => {
  it('should be support slots content', () => {
    const wrapper = mount(Tooltip, {
      slots: {
        content: <div>test message</div>,
      },
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
