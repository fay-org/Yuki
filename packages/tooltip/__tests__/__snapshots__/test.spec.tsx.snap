// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tooltip should be support slots content 1`] = `
<div class="fect-tooltip">
  <!---->
  <!--teleport start-->
  <!--teleport end-->
</div>
`;
